config/engines.yml

```diff
@@ -12,9 +12,9 @@
 brakeman:
   image: codeclimate/codeclimate-brakeman
   description: Static analysis tool which checks Ruby on Rails applications for security vulnerabilities.
-  community: true
+  community: false
   enable_regexps:
-    - \.rb$
+    - ^script\/rails$
```

Comment:
@wfleming Actually, taking a look at the code, it seems like `upgrade_languages`, if present, serve as an [additional filtering check](https://github.com/codeclimate/codeclimate/blob/master/lib/cc/cli/upgrade_config_generator.rb#L24) on top of `regex` and `files _present`, but that the default will be whatever the [latter](https://github.com/codeclimate/codeclimate/blob/master/lib/cc/cli/config_generator.rb#L57) comes up as.

If that's the case, it seems to me that the additional languages check doesn't add much value, and might cause harm in exceptional cases (e.g. rails repos that get added with `javascript` as default language bc of settings in Github).